libraries/Microsoft.Bot.Builder/VersionInfo.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Bot.Builder
+{
+#if DEBUG
+    public
+#else
+    internal
+#endif
+    class VersionInfo
+    {
```

Comment:
I don't understand this class, or how / where it's used. Consider removing.

Reply:
It's used for channels to gather information about the bot's version information when using streaming extensions.